components/infra-proxy-service/server/orgs.go

```diff
@@ -267,18 +222,22 @@ func fromStorageOrg(s storage.Org) *response.Org {
 		Id:        s.ID.String(),
 		Name:      s.Name,
 		AdminUser: s.AdminUser,
-		AdminKey:  s.AdminKey,
 		ServerId:  s.ServerId,
 		Projects:  s.Projects,
 	}
 }
 
 // Create a response.OrgsList from an array of storage.Org
-func fromStorageToListOrgs(sl []storage.Org) []*response.Org {
-	tl := make([]*response.Org, len(sl))
+func fromStorageToListOrgs(sl []storage.Org) []*response.OrgListItem {
+	tl := make([]*response.OrgListItem, len(sl))
 
 	for i, org := range sl {
-		tl[i] = fromStorageOrg(org)
+		tl[i] = &response.OrgListItem{
```

Comment:
Populate list only for used attributes.

Reply:
I was puzzled why you need both `Org` and `OrgListItem`. As far as I can tell, OrgListItem is used exclusively for the `GetOrgs` call, data which is used to populate the orgs list page, is that correct? Yet for a singular `GetOrg` call you return an `Org`, which is inconsistent.
So is there a reason you could not just return a list of `Org` instead of a list of `OrgListItem` ? In other words, is there a reason not to return the other fields?

(I may be missing any context there; my observations are just from reading the code.)

Reply:
Yes, OrgListItem indented for GetOrgs call, the reason I am putting it because I wanted to exclude data(`credential_id`) that are not required for the Org list page.

I had decided when I was specifically wanted to exclude `admin_kye`, now it is `credential_id`

Reply:
I understand they are not required for the Org list page, but are we trying to actively hide them? (That may be a question for @vj ? ) If we are not actively trying to hide them, seems like getting the smaller payload is a micro-optimization at the cost of more complicated code (i.e. maintaining both Org and OrgListItem).
(Not a big deal either way, I just always opt for less code where possible.)